### DISCUSSION TITLE
Is this plugin still active?

### DISCUSSION THREAD

**Discussion by the1gofer** [mdelobelle/metadatamenu#715] [General]
Just wondering if it is still maintianed, or if I should start transitioning away from it.

**Comment by mdelobelle** [comment]
Hi there, I've quite busy lately.
I'll try to find some time to catch up on main issues

**Reply by LynetteCullens** [reply]
I mourned you....Glad to see that you're still here.

**Comment by the1gofer** [comment]
Hey, you do you.  I just wanted to check.
